### PR TITLE
pm2: add Indonesian translation

### PR DESCRIPTION
**Pull request by Paskalouis**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/common/pm2.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+`pm2 list`
+
+- Pantau semua proses:
+
+`pm2 monit`
+
+- Menghentikan sebuah proses:
+
+`pm2 stop {{myapp}}`
```
Comment (navarroaxel) [inline pages.id/common/pm2.md:21]: I think we can update the English page to use `application_name` instead of `myapp`. Do you want to send the update?

Comment (Paskalouis) [reply pages.id/common/pm2.md:21]: Ok sure, wait

Comment (Paskalouis) [reply pages.id/common/pm2.md:21]: I'll use `nama_aplikasi` for this is Indonesian translation

Comment (Paskalouis) [reply pages.id/common/pm2.md]: Done, please check it. Thanks!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I guess we can update the other pages after this PRs merge or should we update it in this Pr itself?

**Comment by Paskalouis** [comment]
> LGTM, Thanks for your contribution. I guess we can update the other pages after this PRs merge or should we update it in this PR itself?

@kbdharun 
Hi, I think we can update this in this PR. Which other page do you mean that needs to be updated?

**Comment by kbdharun** [comment]
> > LGTM, Thanks for your contribution. I guess we can update the other pages after this PRs merge or should we update it in this PR itself?
> 
>  Hi, I think we can update this in this PR. Which other page do you mean that needs to be updated?

Sure, We need to update this page https://github.com/tldr-pages/tldr/blob/main/pages/common/pm2.md with @navarroaxel's suggestion.

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**